SearchService/src/main/java/BaseEntity.java

```diff
@@ -0,0 +1,47 @@
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import org.hibernate.envers.NotAudited;
+import org.hibernate.search.annotations.IndexedEmbedded;
+
+import javax.persistence.Column;
+import javax.persistence.Id;
+import javax.persistence.MappedSuperclass;
+import java.io.Serializable;
+
+/**
+ *
+ * @author claus.straube
+ */
+@MappedSuperclass
+public abstract class BaseEntity implements Cloneable, Serializable {
+
+    @Column(name = "OID")
+    @Id
+    private Long oid;
```

Comment:
ID entweder mit @GeneratedValue annotieren, oder den IDService ebenfalls in die Lib rein.